litmus-portal/frontend/src/components/Header/styles.ts

```diff
@@ -63,7 +59,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   projectDisplay: {
     marginTop: theme.spacing(-1.25),
     fontSize: 12,
-    color: 'grey',
+    color: theme.palette.projectDisplayColor,
```

Comment:
Can we have a better name for the themes! Like `theme.palette.project.grey`

Reply:
color name we can't put in the variable name because it will change based on the project

Reply:
No I mean if we could just change the name from projectDisplayColor to project Object and have grey assigned to any hex value that you are using in your project.

Like
```
project: {
        grey: '#value',
},
```

Reply:
yes we can do that, but here grey is denoting a color, so let say we need the green color so the value will be `gray: green`, which will create confusion